src/verifai/samplers/multi_objective.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+
+class MultiObjectiveSampler:
+
+    def set_priority_graph(self, graph):
+        self.priority_graph = graph
+
+    """
+    Update distribution from list of objectives.
```

Comment:
Docstring should be inside function definition. Also should not assume you're using `ScenicSampler`.